packages/framework/src/Resources/views/Debug/Elasticsearch/template.html.twig

```diff
@@ -0,0 +1,139 @@
+                </svg>
+            </span>
+            <strong>Elasticsearch</strong>
+            <span class="count">
+                <span>{{ collector.data.requestsCount }}</span>
+            </span>
+        </span>
+    {% endif %}
+{% endblock %}
+
+{% block panel %}
+    <style type="text/css">
+        .status--ok {
+            background-color: #DDFFDD;
+        }
+        .status--404 {
+            background-color: #FFFFDD;
+        }
+        .status--error {
+            background-color: #FFDDDD;
+        }
+    </style>
+
+    <h1>Elasticsearch - total {{ collector.data.requestsCount }} queries in {{ '%0.2f'|format(collector.data.totalRequestsTime * 1000) }} ms</h1>
+
+    {% for index, request in collector.data.requests %}
+        {% if request.statusCode is null or request.statusCode >= 500 %}
+            {% set statusClass = 'status--error' %}
+        {% elseif request.statusCode >= 400 %}
+            {% set statusClass = 'status--404' %}
+        {% else %}
+            {% set statusClass = 'status--ok' %}
+        {% endif %}
+        <table>
+            <tr style="">
+                <th class="{{ statusClass }}" style="width: 28px;">{{ request.statusCode }}</th>
+                <th class="{{ statusClass }}" style="width: 70px;">{{ request.method }}</th>
+                <th class="{{ statusClass }}">{{ request.uri }}</th>
+                <td class="{{ statusClass }}" style="width: 200px;">
+                    {% if request.requestJson is not null %}
+                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-response-data-{{ index }}').style.display = ''; return false;">
```

Comment:
what about toggle functionality?

Reply:
Yes I wanted toggle, but plain Javascript is too long and I didn't want a lot of JS in template.
However, I am agree to any improvements.

Reply:
so what about this?
```diff
-                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-response-data-{{ index }}').style.display = ''; return false;">
+                        <button type="button" style="float: right;" onclick="document.getElementById('js-elasticsearch-response-data-{{ index }}').classList.toggle('hidden'); return false;">
-                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-request-data-{{ index }}').style.display = ''; return false;">
+                        <button type="button" style="float: right;" onclick="document.getElementById('js-elasticsearch-request-data-{{ index }}').classList.toggle('hidden'); return false;">
-                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-request-json-{{ index }}').style.display = ''; return false;">
+                        <button type="button" style="float: right;" onclick="document.getElementById('js-elasticsearch-request-json-{{ index }}').classList.toggle('hidden'); return false;">
-                <tr id="js-elasticsearch-request-json-{{ index }}" style="display: none;">
+                <tr id="js-elasticsearch-request-json-{{ index }}" class="hidden">
-                <tr id="js-elasticsearch-request-data-{{ index }}" style="display: none;">
+                <tr id="js-elasticsearch-request-data-{{ index }}" class="hidden">
-                <tr id="js-elasticsearch-response-data-{{ index }}" style="display: none;">
+                <tr id="js-elasticsearch-response-data-{{ index }}" class="hidden">
```

Reply:
https://caniuse.com/#feat=classlist

Reply:
excellent, thanks for something new for me :)

Reply:
so please make fixup, or you want me to do it?